src/frontend/packages/lib_markdown/src/components/CodeMirrorEditor/index.spec.tsx

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable jest/valid-expect */
```

Comment:
no-container and no-node-access can be a real pain in the ass to fix but valid-expect should be pretty easy no ?